UE/Application/Sms.cpp

```diff
@@ -0,0 +1,13 @@
+#include "Sms.hpp"
+
+
+namespace ue
+{
+
+Sms::Sms(common::PhoneNumber from,
+                         std::string text):
+    from(from),
+    text(text)
+{}
```

Comment:
The formatting doesn't look right to me
```suggestion
Sms::Sms(common::PhoneNumber from, std::string text)
    : from(from),
      text(text)
{
}
```